connector.go

```diff
@@ -30,6 +30,7 @@ type Connector interface {
 	Call16Typed(functionName string, args interface{}, result interface{}) (err error)
 	Call17Typed(functionName string, args interface{}, result interface{}) (err error)
 	EvalTyped(expr string, args interface{}, result interface{}) (err error)
+	ExecuteTyped(expr string, args interface{}, result interface{}) (SQLInfo, []ColumnMetaData, error)
```

Comment:
It would be better to add missed methods `ExecTyped/ExecAsync` in a separate commit.

Reply:
Done.

Reply:
The commit is a good place for implementation in multi/multi.go  too.

Reply:
Fixed, sorry for inattention.

Reply:
Could you also add a 'fixed' note into the CHANGELOG.md?

Reply:
Sure, added.